sql/hive/src/test/scala/org/apache/spark/sql/hive/UDFSuite.scala

```diff
@@ -32,5 +32,6 @@ class UDFSuite extends QueryTest {
     assert(sql("SELECT RANDOM0() FROM src LIMIT 1").head().getDouble(0) >= 0.0)
     assert(sql("SELECT RANDOm1() FROM src LIMIT 1").head().getDouble(0) >= 0.0)
     assert(sql("SELECT strlenscala('test', 1) FROM src LIMIT 1").head().getInt(0) === 5)
+    assert(sql("SELECT current_database()").head().getString(0) === "default")
```

Comment:
`SELECT` clause without `FROM` HiveQL seems not supported. Can you confirm if it pass the unit test in your local?